package.json

```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/ArrowSphere/nodejs-api-client.git"
   },
-  "version": "3.125.0",
+  "version": "3.125.0-rc.vgj.3",
```

Comment:
Here you should target a prod version, so the latest is currently 3.125.0, It looks like you add a feature / type, so you should increment the second number:
`3.126.0`